README.md

```diff
@@ -378,8 +378,8 @@ to set `true`; it's effectively a shortcut for `foo=true`).
   for code which relies on `Function.length`.
 
 - `keep_fnames` -- default `false`.  Pass `true` to prevent the
-  compressor from mangling/discarding function names.  Useful for code relying on
-  `Function.prototype.name`.
+  compressor from discarding function names.  Useful for code relying on
+  `Function.prototype.name`. See also [the mangle option](#mangle).
```

Comment:
Additional small refinement: 

s/See also [the mangle option]/See also: the `keep_fnames` [mangle option]/